src/lib/components/Navlink.svelte

```diff
@@ -23,6 +23,7 @@
 		color: var(--text);
 		text-decoration: none;
 		font-size: 1rem;
+		transition-delay: 100ms;
```

Comment:
I don't think this is sufficient to have an animation, did you mean for the color to change gradually?